src/box/iproto.cc

```diff
@@ -500,6 +500,19 @@ struct iproto_connection
 	 * connection.
 	 */
 	struct cmsg destroy_msg;
+	/**
+	 * Wait for shutdown readiness and close connection.
+	*/
+	struct ev_async after_shutdown;
```

Comment:
Please, don't leave the commit message empty. I don't understand what is happening here. Just some new random fields and functions, but how do you intend to use them? Why couldn't they be a part of a more "whole" commit? The change is not really atomic, I mean.